notes-desktop-pwa/build/manifest.webmanifest

```diff
@@ -0,0 +1 @@
+{"name":"notes pwa","short_name":"npwa","start_url":"/","display":"standalone","background_color":"#ffffff","lang":"en","scope":"/","theme_color":"#ffffff","icons":[{"src":"pwa-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/pwa-512x512.png","sizes":"512x512","type":"image/png"},{"src":"pwa-512x512.png","sizes":"512x512","type":"image/png","purpose":"any maskable"}]}
```

Comment:
You need to add a `.gitignore` file at the root of your project and add the following to it:
```
# Logs
logs
*.log
npm-debug.log*
yarn-debug.log*
yarn-error.log*
pnpm-debug.log*
lerna-debug.log*

build
node_modules
/*dist
*.local

# Editor directories and files
.vscode/*
!.vscode/extensions.json
.idea
.DS_Store
*.suo
*.ntvs*
*.njsproj
*.sln
*.sw?

```

The purpose of this is to remove `build`, `node_modules` and other directories that are "built" or references external to your code so your repository is clean of extras. Should just be the code necessary "pre-build".